### ISSUE TITLE
OpenStore opens in background when another app is already opened

### ISSUE THREAD

**Issue by sap-nocops** [ubports/unity8#313]
# STEPS TO REPRODUCE

1. open an app
2. open the drawer and tap and hold on whatever app

## EXPECTED BEHAVIOR

OpenStore opens in foreground

## ACTUAL BEHAVIOR

OpenStore opens in background

**Comment by Flohack74** [comment]
No reproducible for atm, can you recheck this with latest OTA?

**Comment by cibersheep** [comment]
It still happens for me in Arale on the latest dev

**Comment by Flohack74** [comment]
App Drawer: Tap and hold does not do anything
Launcher: If I tap and hold I get a popup for removing the app from the launcher

**Comment by sap-nocops** [comment]
it should open the open store if is a click, #181 is pr that introduced this behaviour. I broke my OPO and cannot test it right now. I will try on my pinephone as soon as I can

**Comment by cibersheep** [comment]
Tap and hold on an app from the Drawer, should open the OpenStore to see details or remove.


![taphold](https://user-images.githubusercontent.com/6640041/100113792-ff566900-2e70-11eb-83c8-d8fce6e1a29f.gif)

**Comment by Flohack74** [comment]
Ok gotcha this is a bug ^^

**Comment by sap-nocops** [comment]
yes I can reproduce it on the pinephone too. I also noticed that if the openstore is already opened in the background the behaviour is as expected
